tokio-util/src/sync/cancellation_token.rs

```diff
@@ -275,6 +278,13 @@ impl CancellationToken {
         }
     }
 
+    /// Returns a `DropGuard` for this token.
+    /// This guard will cancel this token (and all its children)
+    /// on drop unless disarmed.
```

Comment:
Newlines in doc-comments do not translate to newlines in the generated documentation unless there is an empty line.

Reply:
Yeah, i just dislike long lines :)

Do you think splitting first sentence into a separate paragraph makes sense?

Reply:
Sure.

Generally, we wrap documentation at 80 chars to avoid long lines.